.github/pull_request_template.md

```diff
@@ -2,13 +2,17 @@
 Thanks for contributing to Backpack :pray:
 
 Please include a description of the changes you are introducing and some screenshots if appropriate.
+
+Please ensure your pull request title is clear as it will be used to generate the changelog.
+
+Add `major`, `minor` or `patch` label depending on the change according to [semver](semver.org) or `skip-changelog` if the change shouldn't be added to the changelog (e.g. a change to a test or documentation)
```

Comment:
there's a check to ensure the label was added, but also mentioned it here to avoid confusion